config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -47,18 +50,20 @@ teams:
   contributor-comms:
     description: alert members of new issues labeled with contributor comms
     maintainers:
+    - mfahlandt
+    - kaslin
+    - chris-short
     members:
     - AvineshTripathi
-    - chris-short
     - fsmunoz
-    - kaslin
     - sandipanpanda
     - SD-13
     privacy: closed
   contributor-site-admins:
     description: admin access to contributor-site
     maintainers:
     - mrbobbytables
+    - mfahlandt
```

Comment:
Please add yourself to members list. `maintainers` list is only for @kubernetes/owners.